tron/kubernetes.py

```diff
@@ -225,6 +225,8 @@ def get_runner(self, kubeconfig_path: str, queue: PyDeferredQueue) -> Optional[S
             return self.runner
 
         try:
+            # TODO(TRON-1701): we'll need to figure out a good way to support multiple clusters here
```

Comment:
i have some ideas on how to do this now, but while i'm still working on this code i can just do the "dumb" thing and muck with tron code locally to override the config here to what i actually need for spark